conode/conode.go

```diff
@@ -22,6 +22,7 @@ import (
 	_ "github.com/dedis/cothority/cosi/service"
 	_ "github.com/dedis/cothority/guard/service"
 	_ "github.com/dedis/cothority/identity"
+	_ "github.com/dedis/cothority/pop/service"
```

Comment:
After discussion we will merge `pop` back into master, but the website for `33c3` which was quite hacky (but running!) will go into the `archives`.